packages/entity-detail/src/util/detailView/reduceInitialValues.js

```diff
@@ -0,0 +1,30 @@
+import _pick from 'lodash/pick'
+import _isObject from 'lodash/isObject'
+
+export const getRelevantAttributes = modelType => {
```

Comment:
that does not really make sense since almost no other type is a object.
i thought more about something like this:

```
const typeTransformers = {
retlation: (value, model)=> {
 //do stuff
return transormedValue
},
string: (value, model) => {
return value + "!!" //example that appends !! to all strings
}

}


//and then:

typeTransformers[fieldModel.type](value, fieldModel)
```

Reply:
Could you clarify your comment? `modelType` is always a string in my solution, not an object.